raiden/api/rest.py

```diff
@@ -84,8 +95,8 @@ def add_resource(self, resource_cls):
             resource_cls._route
```

Comment:
> Ofcourse if you have a specific suggestion please go ahead.

Just not have a private attribute that is externally accessed. `add_resource('/channels', ChannelsResource)` will do just as fine without any confusion